pool_recycle/plugin.py

```diff
@@ -67,8 +67,7 @@ def get_nodes(self):
         pool_nodes = []
         if 'nodes' in docker_nodes and docker_nodes['nodes'] is not None:
             for node in docker_nodes['nodes']:
-                if ('pool' in node['Metadata'] and
-                   node['Metadata']['pool'] == self.pool):
+                if ('pool' in node['Metadata'] and node['Metadata']['pool'] == self.pool or 'tsuru.io/pool' in node['Metadata'] and node['Metadata']['tsuru.io/pool'] == self.pool):
```

Comment:
The tsuru API already exposes a node's pool as a node attribute, there's no need to read it from the metadata map. This code can simply be `if node.get('Pool') == self.pool: ...`